proofread and corrected some typos / formatting

erikrols opened the pull request.

erikrols committed:
proofread and corrected some typos / formatting

jgarcows commented on Lab 3/README.md:
```diff
@@ -54,11 +54,11 @@ In `watson-deployment.yml`, update the image tag with the registry path to the i
 # Create a bluemix service via the cli
 
-In order to begin using the watson tone analyzer (the bluemix service this application), we must first request an instance of the analyzer in the org and space we have set up our cluster in. If you need to check what space and org you are currently using, simply run `bx login` and select the space and org you were using for stage 1 and 2 of the lab.
+In order to begin using the watson tone analyzer (the bluemix service for this application), we must first request an instance of the analyzer in the org and space we have set up our cluster in. If you need to check what space and org you are currently using, simply run `bx login` and select the space and org you were using for stage 1 and 2 of the lab.
```
We should mention how `bx target --cf` also needs to be run after `bx login` and that's where the org and space are actually selected.

jgarcows commented:
@erikrols Can you add something about the `bx target --cf`   I'll merge the pull request when done

erikrols committed:
Update README.md

added "bx target --cf" and some more words to clarify the approach - @jgarcows pls check.
